avocado/utils/project.py

```diff
@@ -0,0 +1,17 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Author: Lucas Meneghel Rodrigues <lookkas@gmail.com>
+"""
+This module provides shared avocado project information
+"""
+
+NEW_ISSUE_LINK = 'https://github.com/avocado-framework/avocado/issues/new'
```

Comment:
Well, this is tricky. Ideally we want to split `avocado.utils` from `avocado.core` and the `avocado.utils` should not contain any knowledge of avocado (it might contain reference to itself, therefor the URL is right). Anyway for `job` purpose you can perhaps use `avocado.core.defaults` which stores some defaults and this IMO fits as well.

Reply:
Sure, this was mostly an idea. I'm happy to drop those patches.